### ISSUE TITLE
Mail notification as spam (Google Apps for business)

### ISSUE THREAD

**Issue by tdutrion** [phpmentoring/testapp#46]
I just had a notification this night, and the email went straight to my spam folder on Gmail (Google Apps for business account).

![phpmentoring-spam](https://cloud.githubusercontent.com/assets/1712222/9288337/959c3daa-4339-11e5-9878-9512c0d7386b.png)

Here is the link from the yellow notification: [Google support: why is this message in spam](https://support.google.com/mail/answer/1366858?hl=en-GB&authuser=1&expand=5)

Unfortunately, looking at the email headers I'm not able to see what's wrong here...

```
Delivered-To: ****@****.com
Received: by 10.37.57.193 with SMTP id g184csp48836yba;
        Fri, 14 Aug 2015 17:34:48 -0700 (PDT)
X-Received: by 10.55.215.83 with SMTP id m80mr4745758qki.98.1439598888624;
        Fri, 14 Aug 2015 17:34:48 -0700 (PDT)
Return-Path: <no-reply@phpmentoring.org>
Received: from web1.phpmentoring.org (web1.phpmentoring.org. [104.236.23.66])
        by mx.google.com with ESMTP id v106si12501557qgd.75.2015.08.14.17.34.48
        for <****@****.com>;
        Fri, 14 Aug 2015 17:34:48 -0700 (PDT)
Received-SPF: pass (google.com: best guess record for domain of no-reply@phpmentoring.org designates 104.236.23.66 as permitted sender) client-ip=104.236.23.66;
Authentication-Results: mx.google.com;
       spf=pass (google.com: best guess record for domain of no-reply@phpmentoring.org designates 104.236.23.66 as permitted sender) smtp.mailfrom=no-reply@phpmentoring.org
Received: from app.phpmentoring.org (localhost [127.0.0.1])
    by web1.phpmentoring.org (Postfix) with ESMTP id D5590161B28
    for <****@****.com>; Fri, 14 Aug 2015 20:34:47 -0400 (EDT)
Message-ID: <d9e4fd98fdc337aa68b2c97e1a0f13f4@app.phpmentoring.org>
Date: Fri, 14 Aug 2015 20:34:47 -0400
Subject: You recieved a new message
From: no-reply@phpmentoring.org
To: ****@****.com
MIME-Version: 1.0
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hi ****,

**** sent you a private message on the PHP Mento=
ring website.

Here is the link to view it:

http://app.phpmentorin=
g.org/conversations/*
```

Apparently there is a typo in the subject (`recieved` => `received`) but I don't think it's enough to go to spam...

**Comment by christickner** [comment]
DKIM might be useful to verify the emails are legit: http://blog.fastmail.com/2011/10/12/dkim-signing-outgoing-email-with-from-address-domain/

**Comment by tdutrion** [comment]
Did anyone experienced the same problem? I fixed the typo in 21a90e8, and I guess I received all the notifications after that one, but I did unspam it...

Not sure whether it is still an issue, no one else complained... (or do people not look at their spam folder?)

**Comment by dragonmantank** [comment]
I know I sent a couple to Jeff Carouth after I added the SPF records to the DNS, and he responded fairly quickly. Looking at the postfix logs on the server it doesn't look like we're getting graylisted anymore either, so SPF seems to have handled it.
